surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/JUnit47ConcurrencyIT.java

```diff
@@ -32,14 +32,14 @@
  *
  * @author Kristian Rosenvold
  */
-public class Junit47concurrencyIT
```

Comment:
Why changing these names is important? Are you sure you have changed all occurencies of folder `concurrentjunit47`?